src/main/java/com/google/devtools/build/lib/vfs/bazel/Blake3HashFunction.java

```diff
@@ -25,6 +25,14 @@
 
 /** A {@link HashFunction} for BLAKE3. */
 public final class Blake3HashFunction implements HashFunction {
+  private static Blake3HashFunction blake3HashFunction;
```

Comment:
Could this just be a static final field initialized inline?

Reply:
Oh right, that is indeed much cleaner 👍